iam-login-service/src/main/java/it/infn/mw/iam/api/account/password_reset/PasswordDTO.java

```diff
@@ -25,7 +26,8 @@ public class PasswordDTO {
   private String currentPassword;
   
   @NotEmpty(message = "The password cannot be empty")
-  @Length(min = 5, message = "The password must be at least 5 characters")
+  @Length(min = 8, message = "The password must be at least 8 characters")
+  @Pattern(regexp="^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)(?=.*[@$!%*?&])[A-Za-z\\d@$!%*?&]{8,}$", message = "The password must be strong")
```

Comment:
Could you explain in words the meaning of that REGEXP? (Minimum 8 characters, one capital letter, one symbol, etc). I'd change the message in a way that this list of things is clear to the user.

Reply:
Yes, I fix it